poller/poller.go

```diff
@@ -139,8 +139,11 @@ func poll(pool *pgxpool.Pool, ctx *CosmosCallContext, lastHeight int64) (int64,
 
 func Run(pool *pgxpool.Pool, ctx *CosmosCallContext) {
 	lastHeight, err := getHeight(pool)
-	if err != nil {
+	for err != nil {
 		logger.L.Panicw("Cannot get height from database", "error", err)
+		time.Sleep(sleepInitial)
```

Comment:
`Panicw` calls `panic` (something like `throw` in other languages) so this is meaningless
```Go
lastHeight  := int64(0)
toSleep := sleepInitial
for {
    lastHeight, err := getHeight(pool)
    if err == nil {
        break
    }
    logger.L.Errorw("Cannot get height from database", "error", err)
    time.Sleep(toSleep)
    // exponential back-off with max cap
    toSleep = toSleep * 2
    if toSleep > sleepMax {
        toSleep = sleepMax
    }
}
toSleep = sleepInitial
// ...

```

Reply:
hmm i don't understand what was I encountering in testnet indexer then, let me try to reproduce...

Reply:
cannot reproduce... will close this pr for now